### PR TITLE
Add more EXAMPLES for mr subcommands

### PR DESCRIPTION
**Pull request by maxice8**
**Description**

Add EXAMPLES section for almost all subcommands

**How Has This Been Tested?**

`bin/glab <subcommand> --help`

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation
- [ ] Chore (Related to CI or Packaging to platforms)

### CODE REVIEW

File: commands/mr/close/mr_close.go [hunk 1]
```diff
@@ -18,6 +19,12 @@ func NewCmdClose(f *cmdutils.Factory) *cobra.Command {
 		Short: `Close merge requests`,
 		Long:  ``,
 		Args:  cobra.MaximumNArgs(1),
+		Example: heredoc.Doc(`
+			$ glab mr close 1
+			$ glab mr close branch
+			$ glab mr close username:branch
+			$ glab mr close branch -R another/repo
```
Comment (profclems) [inline commands/mr/close/mr_close.go]: Also without supplying any argument, it finds the mr for the current branch
```suggestion
		Example: heredoc.Doc(`
			$ glab mr close   # Closes mr for current branch
			$ glab mr close 1
			$ glab mr close branch
			$ glab mr close username:branch
			$ glab mr close branch -R another/repo
```

**Review by profclems — CHANGES_REQUESTED** [review]

File: commands/mr/delete/mr_delete.go [hunk 2]
```diff
@@ -18,7 +19,11 @@ func NewCmdDelete(f *cmdutils.Factory) *cobra.Command {
 		Long:    ``,
 		Args:    cobra.MaximumNArgs(1),
 		Aliases: []string{"del"},
-		Example: "$ glab delete 123",
+		Example: heredoc.Doc(`
+			$ glab delete 123
+			$ glab del 123
+			$ glab delete branch
```
Comment (profclems) [inline commands/mr/delete/mr_delete.go]: You missed the `mr` command here:
```suggestion
			$ glab mr delete 
			#=> deletes mr for current branch

			$ glab mr delete 123
			$ glab mr del 123
			$ glab mr delete branch
```

**Review by profclems — CHANGES_REQUESTED** [review]

File: commands/mr/issues/mr_issues.go [hunk 3]
```diff
@@ -20,7 +21,10 @@ func NewCmdIssues(f *cmdutils.Factory) *cobra.Command {
 		Long:    ``,
 		Aliases: []string{"issue"},
 		Args:    cobra.MaximumNArgs(1),
-		Example: "$ glab mr issues 46",
+		Example: heredoc.Doc(`
+			$ glab mr issues 46
+			$ glab mr issues branch
```
Comment (profclems) [inline commands/mr/issues/mr_issues.go:26]:
```suggestion
			$ glab mr issues 46
			$ glab mr issues branch
			$ glab mr issues 
```

**Comment by profclems** [comment]
@maxice8  For all the `mr` subcommands, except `create` and `list`, if the user does not specify a branch or ID, `glab` finds the merge request for the current branch

**Comment by maxice8** [comment]
Updated, thanks for noticing the mistakes.

**Review by profclems — APPROVED** [review]
Thanks a lot for this

**Pull request merged**